atlp-rwanda/winners-c8-bn-be
# #182210052 make trip request

#### What does this PR do?
- Add Trip Request Feature

#### Description of Task to be completed?
- A requester(user) should be able create a trip request
- A requester should be able to view all of his/her trip requests
- A manager should be able to view all trip requests of all his Direct employee
- A requester should be able to delete or update  trip request if it still has a pending status
    
#### How should this be manually tested?
- Make sure there are accommodations available in database
- Using swagger docs(web) to login and create trip requests

#### Any background context you want to provide?
none

#### What are the relevant pivotal tracker stories?
182210052
